dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -358,10 +358,14 @@ $small-footer-standard-width: 400px;
   position: absolute;
   bottom: 0;
   left: 0;
-  width: 625px;
+  width: 650px;
   max-width: 50%;
   min-width: 300px;
-  padding: 1em;
+  padding-bottom: 0.5em;
+}
+
+#copyright-scroll-area {
+  padding: 0.8em;
```

Comment:
It would be nice if copyright-scroll-area at least was in JSX so that we're not introducing another hybrid. Would also be nice if copyright-flyout were not a hybrid, but I would be a little worried that I had some good reason to do it this way.